content/docs/testing.md

```diff
@@ -7,34 +7,34 @@ redirect_from:
 next: testing-recipes.html
 ---
 
-You can test React components similar to testing other JavaScript code.
+Anda dapat men-_testing_ komponen React mirip dengan men-_testing_ kode JavaScript lainnya.
 
-There are a few ways to test React components. Broadly, they divide into two categories:
+Ada beberapa cara untuk men-_testing_ komponen pada React. Secara umum, dibagi menjadi dua kategori:
```

Comment:
```suggestion
Ada beberapa cara untuk mengetes komponen pada React. Secara umum, dibagi menjadi dua kategori:
```

Reply:
rekomendasi pemilihan kata yang lebih nyaman dibaca menurut saya:

dibagi -> terbagi

Reply:
done, thank you.